<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>meshBox</class>
 <widget class="QWidget" name="meshBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>673</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="meshGroup">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>1</y>
     <width>191</width>
     <height>501</height>
    </rect>
   </property>
   <property name="title">
    <string>Mesh</string>
   </property>
   <widget class="QLineEdit" name="fileNameLine">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="loadButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Load</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>50</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="meshInfoBox" name="infoBox" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>171</width>
      <height>407</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="animationWidget" name="animWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>506</y>
     <width>191</width>
     <height>157</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>meshInfoBox</class>
   <extends>QWidget</extends>
   <header location="global">GUI/meshinfobox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>animationWidget</class>
   <extends>QWidget</extends>
   <header location="global">GUI/widgets/animationwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
